File: python_algo/baekjoon/11651.py
N = int(input())
def sol(n):
    arr = []
    for i in range(n):
        arr.append(list(map(int, input().split())))
    arr.sort(key = lambda x: (x[1], x[0]))
    return arr
res = sol(N)
for i in res:
    print(i)

File: Swift_Algorithm/Files/11651.swift
//
//  11651.swift
//  Swift_Algorithm
//
//  Created by Gaeng on 2021/06/07.
//
var arr: [(Int, Int)] = []
(1...Int(readLine()!)!).forEach{ _ in
    let input = readLine()!.split{$0==" "}.map{Int(String($0))!}
    arr.append((input[1], input[0]))
}
for o in arr.sorted(by: {$0 < $1}) {
    print(o.1, o.0)
}
